runtime/src/lib.rs

```diff
@@ -536,11 +537,21 @@ impl EraPayout<Balance> for NoInflationIfNoEras {
             // PoA mode: no inflation.
             (0, 0)
         } else {
-            <pallet_staking::ConvertCurve<RewardCurve> as EraPayout<Balance>>::era_payout(
-                total_staked,
-                total_issuance,
-                era_duration_millis,
-            )
+            // rescale the era payout according to the damping factor while keeping the invariant
+            // era_payou + rest = max_payout
```

Comment:
By keeping this invariant, the damping factor is only reducing the payout to the validator, the rest, which increases if damping factor is less than 100% goes to the nominators. So we are not reducing the inflation only payout amount to validators. Is this the intention?
If we wish to reduce the overall inflation, then the max_payout needs to be scaled as well.

Reply:
Sorry my mistake, the staking pallet will send the "remainder" (rest) amount to be handled by `RewardRemainder` trait (not for nominators). We have set this trait as empty tuple `()`, so the amount will be discarded.